sdk/src/asset_handlers/sfnt_io.rs

```diff
@@ -806,21 +710,43 @@ impl SfntDirectory {
 /// values precede those with greater enum values; order within a given group
 /// of chunks (such as a series of `Table` chunks) must be preserved by some
 /// other mechanism.
-#[derive(Debug, Eq, PartialEq)]
+#[derive(Eq, PartialEq)]
 pub(crate) enum ChunkType {
     /// Whole-container header.
     Header,
     /// Table directory entry or entries.
-    Directory,
+    _Directory,
```

Comment:
Why the `_` here? Seems like you're active matching against it, so it's not ignored?

Reply:
Rust's complaint, specifically, is that `_Directory` is never actually _constructed_, which means it's dead code -- which is true. I'm still looking at my flowchart of the calls into `sfnt_io` module, and how we support them. We Chunk-ify the font in two paths, currently, this one and another, _unused_ one, AFAICT, for `get_box_map`.